nengo_bones/templates/base_script.sh.template

```diff
@@ -9,11 +9,9 @@ STATUS=0  # used to exit with non-zero status if any command fails
 # shellcheck disable=SC2034
 SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 
-if [[ -n "$SCRIPT" ]]; then
-    export JOB_NUMBER="$GITHUB_RUN_ID-$GITHUB_JOB-$SCRIPT-$GITHUB_RUN_ATTEMPT"
-else
-    export JOB_NUMBER="$GITHUB_RUN_ID-$GITHUB_JOB-$GITHUB_RUN_ATTEMPT"
-fi
+export JOB_NUMBER="$GITHUB_RUN_ID"
```

Comment:
I'd originally intentionally left that out because I wanted re-running jobs to use the same job number. But looking at how we're using `JOB_NUMBER`, I don't think that's a requirement (and it might even be better where we use it for things like directory names, so we're using a clean directory rather than overwriting the contents of the previous directory). So yeah I think we might as well add in `GITHUB_RUN_ATTEMPT`.